{"uid":"b14b1b49a89b0a","name":"LoginUsuarioIncorrecto","fullName":"login.LoginTests.LoginUsuarioIncorrecto","historyId":"1af352cb28d70bfd1c81086e74664d0b","time":{"start":1744683201708,"stop":1744683201709,"duration":1},"description":"Verifica que aparezca un mensaje de error al ingresar un nombre de usuario incorrecto","descriptionHtml":"<p>Verifica que aparezca un mensaje de error al ingresar un nombre de usuario incorrecto</p>\n","status":"skipped","statusMessage":"❌ Se detiene ejecución por fallo previo.","statusTrace":"org.testng.SkipException: ❌ Se detiene ejecución por fallo previo.\r\n\tat base.BaseTest.verificarFalloAnterior(BaseTest.java:15)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)\r\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:578)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:819)\r\n\tat org.testng.TestRunner.run(TestRunner.java:619)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1744683194245,"stop":1744683201034,"duration":6789},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"verificarFalloAnterior","time":{"start":1744683201704,"stop":1744683201705,"duration":1},"description":"","status":"broken","statusMessage":"❌ Se detiene ejecución por fallo previo.","statusTrace":"org.testng.SkipException: ❌ Se detiene ejecución por fallo previo.\r\n\tat base.BaseTest.verificarFalloAnterior(BaseTest.java:15)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)\r\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:578)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:819)\r\n\tat org.testng.TestRunner.run(TestRunner.java:619)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\r\n\tat org.testng.TestNG.run(TestNG.java:1101)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"package","value":"login.LoginTests"},{"name":"testClass","value":"login.LoginTests"},{"name":"testMethod","value":"LoginUsuarioIncorrecto"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"selenium-IntelliJIDEA"},{"name":"subSuite","value":"login.LoginTests"},{"name":"host","value":"DESKTOP-MNG9D3P"},{"name":"thread","value":"21100@DESKTOP-MNG9D3P.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Usuario incorrecto"},{"name":"epic","value":"Login"},{"name":"feature","value":"Validaciones del formulario de inicio de sesión"},{"name":"severity","value":"critical"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[{"uid":"ffaeb37649e51115","status":"skipped","statusDetails":"❌ Se detiene ejecución por fallo previo.","time":{"start":1744683040742,"stop":1744683040742,"duration":0}}],"categories":[],"tags":[]},"source":"b14b1b49a89b0a.json","parameterValues":[]}